packages/react-components/react-utilities/src/ssr/SSRContext-node.test.tsx

```diff
@@ -4,7 +4,8 @@
 
 // 👆 this is intentionally to test in SSR like environment
 
-import { renderHook } from '@testing-library/react-hooks';
```

Comment:
cannot use `TL/react` for the `renderhook` api as that required DOM, which is ecplicitly disabled for this test -> thus switching to custom alternative